static/js/social-media.js

```diff
@@ -29,6 +29,12 @@ var SocialMedia = function() {};
       html: "<a href='https://twitter.com/share' class='twitter-share-button' data-count='none' data-text='Check out my OpenBadges at ' data-via='OpenBadges' data-dnt='true'>Tweet</a>"
     },
 
+    linkedin: {
+      id: "linkedin",
+      src: "//platform.linkedin.com/in.js",
+      html: "<script type='IN/Share' data-url='"+urlPlaceHolder+"' data-counter='right'></script>"
```

Comment:
Yeah, this is part of the script snippet provided by linkedin, it has been tested and it functioned as expected.